wandb/util.py

```diff
@@ -1643,6 +1643,14 @@ def _is_artifact_string(v: Any) -> bool:
     return isinstance(v, str) and v.startswith("wandb-artifact://")
 
 
+def _is_artifact_dict(v: Any) -> bool:
```

Comment:
nit: maybe call this `_is_artifact_version_weave_dict` or something really explicit